### DISCUSSION TITLE
The API is down

### DISCUSSION THREAD

**Discussion by danaugrs** [planetscale/discussion#63] [Bug Reports] [answered]
![image](https://user-images.githubusercontent.com/5261448/124970842-d0a39200-dffe-11eb-89e6-97634bce3bdb.png)

**Comment by danaugrs** [comment, marked as answer]
It's ~~back up 🎉~~ intermittent.

**Reply by jgreet** [reply]
Sorry about that! Database connections and the PlanetScale dashboard were unavailable for few minutes today as we recovered from a bad deploy.

**Comment by bigint** [comment]
Also seems like some of the backups are failed too

![image](https://user-images.githubusercontent.com/69431456/125022034-7572ad80-e099-11eb-941f-07e98cad9130.png)

**Reply by jgreet** [reply]
Yes, unrelated to the outage mentioned above we had some trouble with backup jobs erroring for certain databases earlier in the week. They should be back in working order but please do [let us now](https://www.planetscale.com/support) if you're still seeing failures.
